homeassistant/components/tplink_eap/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "TP-Link EAP",
```

Comment:
Does the title need to be translated? It is a brand/product name, I guess that cannot be translated into languages right?

Reply:
Correct, I removed it from the file